### PR TITLE
New loadout items

### PR DESCRIPTION
**Pull request by PopNotes**
:cl: PopNotes
add: More loadout items!
del: Removed loadout masks!
/:cl:

I can already taste the river of all caps crybaby salt from this, so let me egsblain myself.

The clown/mime masks are something sought after, something many would try to get their hands on since there was really only one on the station attached to the clown's face. It kind of loses its meaning when literally everyone and their brother has one at round start. Also, the idea of FREE FACE-CONCEALING GAS MASKS is probably not good.

Oh yeah, and besides that, more loadout items is good in general. Assuming they're either fluff or do so little it's barely worth worrying about, like matches so you can actually light your cigarettes.

### CODE REVIEW

**Comment by Thelolswat** [comment]
>>>>>>[][][]<><><\|\|>>{removes clown mask}

How to remove someone else’s PR
we need our mask

**Comment by Dudeinyourbackyard** [comment]
sure thing
+1

**Comment by Chnkr** [comment]
-1 Don't touch the clown/mime masks

**Comment by McBawbaggings** [comment]
If everyone has access to the clown mask roundstart, then it's not the clown mask
+1

File: hippiestation/code/modules/client/loadout/neck.dm [hunk 1]
```diff
@@ -79,4 +79,7 @@
 	category = slot_neck
 	path = /obj/item/clothing/neck/stripedgreenscarf
 
-
+/datum/gear/headphones
```
Comment (KayeArray) [inline hippiestation/code/modules/client/loadout/neck.dm:82]: yuck

Comment (PopNotes) [reply hippiestation/code/modules/client/loadout/neck.dm]: why yuck

Comment (KayeArray) [reply hippiestation/code/modules/client/loadout/neck.dm:82]: just yuck

Comment (PopNotes) [reply hippiestation/code/modules/client/loadout/neck.dm]: this tells me nothing

Comment (KayeArray) [reply hippiestation/code/modules/client/loadout/neck.dm:82]: it's not supposed to convey to you anything more than my opinion on the matter

File: hippiestation/code/modules/client/loadout/neck.dm [hunk 2]
```diff
@@ -79,4 +79,7 @@
...
+	name = "Headphones"
+	category = slot_neck
```
Comment (ghost) [inline hippiestation/code/modules/client/loadout/neck.dm:84]: why are they going to the neck if they're an ears item?

Comment (PopNotes) [reply hippiestation/code/modules/client/loadout/neck.dm]: They fit on multiple slots.

**Comment by PopNotes** [comment]
I'm pretty sure I'm allowed to merge my own PRs so long as someone else reviews the code and I get enough score with the headmins. Yell at me if that's wrong for some reason.

**Comment by banthebantz** [comment]
# REEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEE

**Comment by PopNotes** [comment]
I am just a man, I do the things I can

**Comment by DinduStuffin** [comment]
I dunno, I agree with removing clown masks, I liked them more when they were coveted rather than something everyone has.

+1

**Pull request merged**